common/containers/Tabs/Contracts/components/Interact/components/InteractForm/index.tsx

```diff
@@ -76,6 +77,48 @@ class InteractForm extends Component<Props, State> {
     return contracts && contracts.length;
   };
 
+  public componentDidMount = () => {
+    setTimeout(
```

Comment:
It would be a good idea to make sure that this timeout is also cleared in `componentWillUnmount`

Reply:
Sorry for the late response, i was travelling! Thanks for taking the time to review the code. What do you mean by timeout is cleared ?

Reply:
Oooh i got it. Sorry for the delay. Thanks for the good feedback